scene/gui/color_picker.cpp

```diff
@@ -1993,6 +1993,10 @@ void ColorPicker::_bind_methods() {
 	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_normal, "tab_unselected", "TabContainer");
 	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_pressed, "tab_selected", "TabContainer");
 	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_hover, "tab_selected", "TabContainer");
+
+	ADD_CLASS_DEPENDENCY("LineEdit");
+	ADD_CLASS_DEPENDENCY("MenuButton");
+	ADD_CLASS_DEPENDENCY("PopupMenu");
```

Comment:
How do you determine which classes should be added as dependencies?

I see `color_picker.cpp` has an internal LineEdit but there's also e.g. SpinBox, HSlider, etc.

Is this a requirement only when signals are used?

Reply:
> Is this a requirement only when signals are used?

Seems so, as connecting signals require the class to be registered.

Reply:
FYI @KoBeWi. I wonder if we could find a more elegant way to do this so we don't need to hardcode such list only for internal classes that have signal connections.

It's fine for now for this PR though.